samples/matter/template/src/template.zap

```diff
@@ -1,5 +1,5 @@
 {
-  "featureLevel": 96,
+  "featureLevel": 98,
```

Comment:
Make sure that you have used the ZAP Tool version that we currently recommend. Otherwise, people will struggle to open this file with the recommended version (I did that in the past ;)).

Reply:
Oh, right, I have two ZAP Tools installed and probably I used the wrong one. Thanks for spotting that!

Reply:
Fixed.